opencti-platform/opencti-front/src/private/components/analyses/reports/ReportCreation.tsx

```diff
@@ -269,6 +269,9 @@ export const ReportCreationForm: FunctionComponent<ReportFormProps> = ({
           <ObjectMarkingField
             name="objectMarking"
             style={fieldSpacingContainerStyle}
+            onChange={() => {
+            }}
```

Comment:
this prop can be removed I suppose, as the callback does nothing.

Reply:
In fact, adding this empty callback "fix" the issue.
And it is the `setFieldValue` that is not needed.

Nonetheless it should not be fixed this way. The real issue is in the `ObjectMarkingField` component where the check on markings is not done if the props `onChange` is undefined.

Reply:
It's a quick fix found to enable the pop-up display when the max-markings are merged. We did what was most urgent.

Reply:
In any case this solution is working only on report creation so not sufficient because it is the same on every creation form. It will be less tedious to implement the real fix than apply this work around on every creation form